standards/webhooks.md

```diff
@@ -0,0 +1,48 @@
+# Webhooks
+
+## Overview
+
+Webhooks are automated messages sent from apps when something happens, typically used to integrate two disconnected systems together, such as with a third-party provider. They have a message—or payload—and are sent to a unique URL configured by the customer or user. Webhooks are an advantage in an API as they are almost always faster than polling for an updated status or parsing to determine which events or changes are new within a timeline. 
```

Comment:
I like this definition a bit better for first sentence. 

"A webhook is an HTTP-based callback function that allows lightweight, usually event-driven communication between two APIs. Webhooks are used by a wide variety of web apps to receive small amounts of data from other apps when specific event occur."